app/src/androidTest/java/org/simple/clinic/medicalhistory/MedicalHistoryRepositoryAndroidTest.kt

```diff
@@ -64,7 +64,7 @@ class MedicalHistoryRepositoryAndroidTest {
         historyEntry = historyEntry
     ).blockingAwait()
 
-    val savedHistory = repository.historyForPatientOrDefault(patientUuid).blockingFirst()
+    val savedHistory = dao.historyForPatientImmediate(patientUuid)!!
```

Comment:
You can use `repository.historyForPatientOrDefaultImmediate` instead of using the `dao` method.

Reply:
I intentionally chose this because the I want to get rid of the `OrDefault` part since it causes issues with maintenance. My eventual goal is move this creation of a default history into the business logic out of the repository.